x/auth/ante.go

```diff
@@ -122,18 +142,21 @@ func NewAnteHandler(ak AccountKeeper, supplyKeeper types.SupplyKeeper, sigGasCon
 			signerAccs[0] = ak.GetAccount(newCtx, signerAccs[0].GetAddress())
 		}
 
+		if len(stdTx.FeeAccount) != 0 {
+			if err := feeAccount.SetSequence(feeAccount.GetSequence() + 1); err != nil {
```

Comment:
Don't think its necessary to update `Sequence` of FeeAccount. It didn't sign anything

Reply:
Right!